frontend/src/pages/Applicant/HomePage.tsx

```diff
@@ -1,52 +1,58 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import { ButtonVariant } from "../../utils/types";
+import { ApplicationFormStatus, ButtonSize, ButtonVariant } from "../../utils/types";
 import Button from "../../components/ui/Button";
+import CohortInfo from "../../components/ui/CohortInfo";
+import ApplicationStatus from "../../components/ui/ApplicationStatus";
+import { deadLineExceededInfo, noOpenApplicationInfo } from "../../utils/data";
+import { useGetFormForApplicantsQuery } from "../../features/user/apiSlice";
+import { applicationStatusHandler, getJWT } from "../../utils/helper";
+import Loader from "../../components/ui/Loader";
 
 const HomePage = () => {
+  const jwt: string = getJWT();
+  const { data: applicationForm, isLoading } =
+    useGetFormForApplicantsQuery(jwt);
```

Comment:
`useGetFormForApplicantsQuery` is a bit misleading because it actually gets you an application form for one applicant (Logged in applicant) with responses (if any are saved).

I'd suggest renaming it to something like `useGetMyApplicationQuery`